### PR TITLE
Player scores

### PR DESCRIPTION
**Pull request by bumblebot1**

### CODE REVIEW

File: src/client/user/user-list.js [hunk 1]
```diff
@@ -5,15 +5,17 @@ import config from '../config/config';
 import style from '../common/common.scss'
 
 export default function UserList(props) {
```
Comment (kenneth-gray) [inline src/client/user/user-list.js]: { users }

File: src/client/user/user-list.js [hunk 2]
```diff
@@ -22,3 +24,24 @@ export default function UserList(props) {
...
+    if(score !== undefined){
+        return (
+            <span className='badge'>{score}</span>
+        );
+    }
+    return null
+}
+
+function sort(users) {
+    users.sort((a, b) => {
```
Comment (kenneth-gray) [inline src/client/user/user-list.js]: a, b?

File: src/client/user/user-list.js [hunk 3]
```diff
@@ -22,3 +24,24 @@ export default function UserList(props) {
...
+        let score = 0-(a.data.score-b.data.score);
+        if(a.score !== undefined && b.data.score !== undefined && score !== 0)
```
Comment (kenneth-gray) [inline src/client/user/user-list.js]: Probably should stick with brackets. In this case you only need the if part and then the else can be outside the if block

File: src/client/user/user-list.js (hunk 1 above)
Comment (JamieZafar) [inline src/client/user/user-list.js]: what do you mean?

**Comment by JamieZafar** [comment]
:anchor: ?

**Comment by thullSL** [comment]
:shipit:

**Pull request merged**